### PR TITLE
Add an extendable template above the method of a controller.

### PR DESCRIPTION
**Pull request by gillesdb**
Hello,

I'm using the SwaggerToCSharpControllerGenerator and I would like to add custom annotations above my methods.
In order to accomplish this, I defined an additional extendable template ''Controller.Method.Annotations.liquid".

In this template, I'm able to define my own custom policies for each method:  
[Policy.{{ operation.ActualOperationName }}()]

 In code:

MicrosoftTeams-image (3)

 I was wondering if this change could be part of the public repo?
Thank you in advance!

### CODE REVIEW

File: src/NSwag.CodeGeneration.CSharp/Templates/Controller.liquid [hunk 1]
```diff
@@ -94,6 +95,7 @@ public abstract class {{ Class }}ControllerBase : {% if HasBaseClass %}{{ BaseCl
 {%         if operation.IsDeprecated -%}
     [System.Obsolete]
 {%         endif -%}
+	{% template Controller.Method.Annotations %}
```
Comment (RicoSuter) [inline src/NSwag.CodeGeneration.CSharp/Templates/Controller.liquid]: Whitespace: Use spaces and align with method

File: src/NSwag.CodeGeneration.CSharp/Templates/Controller.liquid [hunk 2]
```diff
@@ -56,6 +56,7 @@ public partial class {{ Class }}Controller : {% if HasBaseClass %}{{ BaseClass }
...
```
Comment (RicoSuter) [inline src/NSwag.CodeGeneration.CSharp/Templates/Controller.liquid]: Whitespace: Use spaces and align with method

**Comment by RicoSuter** [comment]
Looks good to me. See comments

**Comment by RicoSuter** [comment]
Ready?

**Comment by gillesdb** [comment]
Yes, looks good for me.

**Comment by RicoSuter** [comment]
Thanks for the PR!

**Comment by RicoSuter** [comment]
Will be in v13 which might still take some days.. but there is a preview package & i can publish a preview on nuget.org

**Pull request merged**